tests/test_lints.rs

```diff
@@ -4,6 +4,17 @@ use thiserror::Error;
 
 pub use std::error::Error;
 
+#[test]
+fn test_allow_attributes() {
```

Comment:
This test doesn't actually do anything. For some reason `#![deny(clippy::allow_attributes)]` never works here with any lint - it's as if it's not set. Nor does passing `-Dclippy::allow_attributes`. Maybe something about being a test target for the crate itself? I don't know